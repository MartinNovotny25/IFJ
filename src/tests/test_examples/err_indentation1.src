# neocekavana zmÄ›ny urovne odsazeni (chyba 2)
print('ahoj')
  print('svete')#konec_souboru_EOF