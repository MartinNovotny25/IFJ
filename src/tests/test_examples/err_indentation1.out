TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, ahoj)
TOKEN())
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, svete)
TOKEN())
TOKEN(DEDENT)
TOKEN(EOF)
